Import/Esri/ArcGISRuntime/Toolkit/Controls/CppApi/ArcGISCompass.qml

```diff
@@ -4,8 +4,45 @@ import QtQuick.Controls 1.4
 import QtQuick.Controls.Styles 1.4
 import Esri.ArcGISRuntime.Toolkit.CppApi 100.2
 
+/*!
+    \qmltype ArcGISCompass
+    \ingroup ToolArcGISCompass
+    \inqmlmodule Esri.ArcGISRuntime.Toolkit.Controls
+    \since Esri.ArcGISRutime 100.2
+    \brief The user interface for the ArcGIS Compass tool.
+    \sa {ArcGIS Compass}
+
+    \since Esri::ArcGISRuntime 100.2.0
+*/
+
 Item {
+    /*!
+      \qmlproperty real scaleFactor
+      \brief The scale factor used for sizing UI elements.
+
+      Pixel density and screen resolution varies greatly between different
+      devices and operating systems. This property allows your app to specify
+      the width and height of UI elements so that the sizes appear similar
+      (relative to screen size) across devices. Here is an example of how to
+      use this property.
+
+      \code
+         ArcGISCompass {
+            width: 100 * scaleFactor
+            height: 100 * scaleFactor
+            ...
+         }
+      \endcode
+     */
```

Comment:
I think this could be marked as internal. I don't think we expose it on other types.